spec/super_parser_spec.rb

```diff
@@ -0,0 +1,62 @@
+RSpec.describe SuperParser do
```

Comment:
all tests below do not test a thing. The idea is that your tests should check your expectations and the actual output like the balance of the account or description of the transaction.